fix: always use absolute positioning to prevent jumping Orbit slider

DanielRuf opened the pull request:
Before and after a slide is animted its position attribute is changed between `absolute` and `relative` which can produce a jump of the element after a slide change.

Closes https://github.com/zurb/foundation-sites/issues/11082

DanielRuf committed:
fix: always use absolute positioning to prevent jumping Orbit slider

ncoden commented on js/foundation.orbit.js:
```diff
@@ -344,11 +344,10 @@ class Orbit extends Plugin {
           function(){
-            $newSlide.css({'position': 'relative', 'display': 'block'})
-            .attr('aria-live', 'polite');
+            $newSlide.attr('aria-live', 'polite');
```
Slides are hidden when `motion-ui` is not used.

https://github.com/DanielRuf/foundation-sites/blob/dbdae96312bcccd0820d8ee43c088d5d6a1fab14/js/foundation.orbit.js#L153

We should keep `'display': 'block'`;

DanielRuf replied:
But this causes the jump then.

DanielRuf replied:
Ah, just the block property?

ncoden replied:
Yeah, and remove `'position': 'relative'` from line 153. But make some tests (with and without motionUI / `mui` classes), I'm not sure.

DanielRuf committed:
fix: render slide as block element

ncoden commented:
@DanielRuf Tell me when you done your tests or if I should take care of them 😉

DanielRuf commented:
> or if I should take care of them

Some help is really appreciated and would be great.

ncoden committed:
fix: remove position reset of Orbit slides preventing the "full-absolute" new behavior

All Orbit slides should always have an `absolute` positioning to prevent jumping Orbit slider (see #11107). The `'position': 'relative'` reset is no longer necessary here and break the Orbit behavior as inline styles overrides `.orbit-slide` class styles.

See:
* https://github.com/zurb/foundation-sites/pull/11107/commits/dbdae96312bcccd0820d8ee43c088d5d6a1fab14
* https://github.com/zurb/foundation-sites/pull/11107

ncoden commented:
> Yeah, and remove 'position': 'relative' from line 153. But make some tests (with and without motionUI / mui classes), I'm not sure.

I found a bug before figuring out that `'position': 'relative'` was not removed. I took care of this.

> **7daaba5 fix: remove position reset of Orbit slides preventing the "full-absolute" new behavior**
>
> All Orbit slides should always have an `absolute` positioning to prevent jumping Orbit slider (see #11107). The `'position': 'relative'` reset is no longer necessary here and break the Orbit behavior as inline styles overrides `.orbit-slide` class styles.

ncoden reviewed (approved):
> Yeah, and remove 'position': 'relative' from line 153. But make some tests (with and without motionUI / mui classes), I'm not sure.

I found a bug before figuring out that `'position': 'relative'` was not removed. I took care of this.

> **7daaba5 fix: remove position reset of Orbit slides preventing the "full-absolute" new behavior**
>
> All Orbit slides should always have an `absolute` positioning to prevent jumping Orbit slider (see #11107). The `'position': 'relative'` reset is no longer necessary here and break the Orbit behavior as inline styles overrides `.orbit-slide` class styles.

I made some tests and everything seems to work 👍
